isthmus/src/test/java/io/substrait/isthmus/StringFunctionTest.java

```diff
@@ -0,0 +1,66 @@
+package io.substrait.isthmus;
+
+import java.util.List;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+public class StringFunctionTest extends PlanTestBase {
+
+  static List<String> CREATES = List.of("CREATE TABLE strings (c16 CHAR(16), vc32 VARCHAR(32))");
+
+  @ParameterizedTest
+  @ValueSource(strings = {"c16", "vc32"})
+  void charLength(String column) throws Exception {
```

Comment:
I made a very small change here: `char_length` -> `charLength`

Reply:
6 months of *&#@$%! Go and I forgot good ol' Java conventions.  What a shame!  😉